doc/ranger.pod

```diff
@@ -209,6 +209,9 @@ This does not work over ssh, requires certain terminals (tested on "xterm" and
 To enable this feature, install the program "w3m" and set the option
 C<preview_images_method> to w3m.
 
+When using a terminal with a border larger than 0 which is not automatically detected the w3m preview will be misaligned.
+Use the C<w3m_offset> option to manually adjust the image offset. This should be the same value as the terminals' border value.
```

Comment:
terminals' -> terminal's

Reply:
No need to apologize. I just figure it's better I share whatever typos I notice so the end result is as high quality as we can make it : ) Myself, I have plenty of trouble with usage of commas in english.